include/game.h

```diff
@@ -0,0 +1,20 @@
+#ifndef GAME_H
+#define GAME_H
+
+#include "screen.h"
+
+class Game {
+private:
+  // Private Variables
+  Screen* m_screen;
+
+  Game(){}; // Private Default constructor
+public:
+  // Public Constructor
+  Game(Screen* screen);
```

Comment:
Unless the constructor has other logic inside, you should use an [initializer list](http://en.cppreference.com/w/cpp/language/initializer_list).